schema-derivatives-options.graphql

```diff
@@ -1307,6 +1352,9 @@ type Position @entity {
   " Blocknumber the contract is exercised "
   exercisedBlockNumber: BigInt
 
+  " Price of the underlying asset at the time the option was closed (can be null if it gets closed without exercising) "
+  closedPrice: BigDecimal
```

Comment:
is this in USD?

Reply:
yes

Reply:
@corerouter can you change this to closedPriceUSD?

Reply:
Changed at 1c8014a8203907d0c383d658576f26965836ad9b